options/locale/locale_en-US.ini

```diff
@@ -3460,6 +3461,26 @@ runs.no_matching_runner_helper = No matching runner: %s
 
 need_approval_desc = Need approval to run workflows for fork pull request.
 
+variables = Variables
+variables.management = Variables Management
+variables.creation = Add Variable
+variables.none = There are no variables yet.
+variables.delete =  Remove Variable "%s"
+variables.edit_tooltip =  Edit Variable "%s"
+variables.deletion = Remove variable
+variables.deletion.description = Removing a variable is permanent and cannot be undone. Continue?
+variables.description = Variables will be passed to certain actions and cannot be read otherwise.
+variables.name = Name
+variables.value = Value
```

Comment:
Move these two to top-level translation scope please.

Reply:
No, outside any `[section]`, e.g. before the first section.

Reply:
Should we adjust these to the top-level？
![image](https://github.com/go-gitea/gitea/assets/33891828/94d05dda-e3b5-45da-b3ef-61d33b2ae4bc)

Reply:
Definitely yes, but likely in another PR.

Reply:
Thanks, looks good.